tests/propertyBinding/impl/qml/PropertyBindingInterfaceQmlImplementation.qml

```diff
@@ -0,0 +1,51 @@
+/**********************************************************************
+**
+** Copyright (C) 2018 Luxoft Sweden AB
+**
+** This file is part of the FaceLift project
+**
+** Permission is hereby granted, free of charge, to any person
+** obtaining a copy of this software and associated documentation files
+** (the "Software"), to deal in the Software without restriction,
+** including without limitation the rights to use, copy, modify, merge,
+** publish, distribute, sublicense, and/or sell copies of the Software,
+** and to permit persons to whom the Software is furnished to do so,
+** subject to the following conditions:
+**
+** The above copyright notice and this permission notice shall be
+** included in all copies or substantial portions of the Software.
+**
+** THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+** EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+** MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+** NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS
+** BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN
+** ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+** CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+** SOFTWARE.
+**
+** SPDX-License-Identifier: MIT
+**
+**********************************************************************/
+
+import QtQuick 2.0
+import tests.propertybinding 1.0
+
+PropertyBindingInterfaceTestQMLImplementation {
+    id: root
+    Component.onCompleted: console.log("QML implementation is used.");
+
+    comboStr1: cmb
+
+    ComboStructure {
```

Comment:
Why is that called Combo ?

Reply:
its a structure actually, i like to call it combo as it combines few data types together

Reply:
Combining various data types together is what a structure is all about.

Reply:
so would you recommend some better name ? e.g Brend suggested TestStruct.

Reply:
Changed it to PropertiesStructure And committed

Reply:
@jacky309  kindly review